components/molecule/select/src/index.scss

```diff
@@ -0,0 +1,50 @@
+@import '~@schibstedspain/sui-theme/lib/index';
+@import '~@s-ui/react-atom-input/lib/index';
+@import '~@s-ui/react-molecule-input-tags/lib/index';
+@import '~@s-ui/react-molecule-dropdown-option/lib/index';
+@import '~@s-ui/react-molecule-dropdown-list/lib/index';
+
+$base-class: '.sui-MoleculeSelect';
+$class-input: '#{$base-class}-inputSelect';
+
+$z-select-list: $z-navigation !default;
+$mr-select-list-arrow: -34px !default;
+
+#{$base-class} {
+  position: relative;
+
+  .sui-MoleculeDropdownList {
+    position: absolute;
+    z-index: $z-select-list;
+  }
+
+  #{$class-input} {
```

Comment:
In this case, you don't need to grow up the specificity, if you use `&-inputSelect` here, it would output a single selector (`.sui-MoleculeSelect-inputSelect`) instead of a combined selector (`.sui-MoleculeSelect .sui-MoleculeSelect-inputSelect`).